src/main/java/pl/wojciechkostecki/revel/model/Menu.java

```diff
@@ -0,0 +1,26 @@
+package pl.wojciechkostecki.revel.model;
+
+import lombok.Data;
+
+import javax.persistence.*;
+import java.util.HashSet;
+import java.util.Set;
+
+@Entity
+@Table(name = "menu")
+@Data
+public class Menu {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    @Column(name = "id")
+    private Long id;
+
+    private String name;
+
+    @OneToOne(mappedBy = "menu")
+    private Local local;
+
+    @OneToMany(mappedBy = "menu", cascade = CascadeType.ALL, orphanRemoval = true)
+    private Set<MenuItem> menuItems = new HashSet<>();
```

Comment:
Nice, I like the fact that you have used Set. Even i forgot about it. I would've used a List, lol